CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/reporting/GraphReportManager.java

```diff
@@ -47,15 +45,15 @@
     /**
      * Tags that will be ignored (not recorded) by GraphReport instances.
      */
-    private static final Set<String> IGNORED_TAGS;
+    private static final Set<String> TAGS;
```

Comment:
Given the comment here, I'm not this field is required (given the code that used this field has been removed) so I think this can be removed